openapi/components/headers/X-RateLimit-Retry-After.yaml

```
@@ -0,0 +1,4 @@
+description: UTC timestamp after which the rate limit will reset and the request can be retried.
```

suggestion: to avoid future tense
```suggestion
description: UTC timestamp after which the rate limit resets and the request can be retried.
```